internal/pkg/mapper/relations/link_test.go

```diff
@@ -5,103 +5,93 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-	"go.uber.org/zap"
 
 	"github.com/keboola/keboola-as-code/internal/pkg/fixtures"
 	. "github.com/keboola/keboola-as-code/internal/pkg/mapper/relations"
 	"github.com/keboola/keboola-as-code/internal/pkg/model"
-	"github.com/keboola/keboola-as-code/internal/pkg/testapi"
-	"github.com/keboola/keboola-as-code/internal/pkg/testhelper"
 )
 
-func TestRelationsMapperLinkOnObjectsLoad(t *testing.T) {
+func TestRelationsMapperLinkRelations(t *testing.T) {
 	t.Parallel()
-	components := model.NewComponentsMap(testapi.NewMockedComponentsProvider())
-	fs := testhelper.NewMemoryFs()
-	state := model.NewState(zap.NewNop().Sugar(), fs, components, model.SortByPath)
+	context, _ := createMapperContext(t)
 
 	key1 := fixtures.MockedKey{Id: "123"}
 	key2 := fixtures.MockedKey{Id: "456"}
 
-	// ObjectManifest side
-	manifest1 := &fixtures.MockedManifest{MockedKey: key1}
-	object1 := &fixtures.MockedObject{
-		MockedKey: key1,
-		Relations: model.Relations{
-			&fixtures.MockedManifestSideRelation{
-				OtherSide: key2,
+	// Manifest side
+	object1 := &fixtures.MockedObjectState{
+		MockedManifest: &fixtures.MockedManifest{MockedKey: key1},
+		Local: &fixtures.MockedObject{
+			MockedKey: key1,
+			Relations: model.Relations{
+				&fixtures.MockedManifestSideRelation{
+					OtherSide: key2,
+				},
 			},
 		},
 	}
-	objectState1, err := state.GetOrCreateFrom(manifest1)
-	assert.NoError(t, err)
-	objectState1.SetLocalState(object1)
+	assert.NoError(t, context.State.Set(object1))
```

Comment:
Niekedy v minulosti pribudla metoda `state.State.Set`, ale v tychto testoch to nebolo fixnute, tak sa stav objektu vytvaral zlozito.

Fixol som to vo vsetkych testoch